src/components/app/matching/MatchesList.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { Match, MatchProps } from "./Match";
+
+export interface MatchesListProps {
+  matches: Array<MatchProps>
+}
+
+export const MatchesList = (props: MatchesListProps) => {
+
+  function renderMatch(match: MatchProps, listIdx: number) {
+    return <Match key={listIdx} {...props.matches[listIdx]}/>
```

Comment:
accessing the match with `props.matches[listIdx]` is redundant when we already have `match` as a parameter